### ISSUE TITLE
Disease (MONDO:0008753) - related_to - NamedThing  - related_to - Gene (NCBIGene:3081

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/testing#138]
**Query:** [D.1-hgd-alkaptonuria.json](https://github.com/NCATSTranslator/testing/blob/main/ars-requests/not-none/1.2/D.1-hgd-alkaptonuria.json)
**PK:** ef6d4f87-f53c-4f8c-9aba-a804d74c5761
**Control:** looking for processes involving the breaking down of tyrosine and phenylalanine

**Comment by andrewsu** [comment]
Anyone else getting an error here https://arax.ncats.io/?r=ef6d4f87-f53c-4f8c-9aba-a804d74c5761 "Response with id=ef6d4f87-f53c-4f8c-9aba-a804d74c5761 was not found (404)."

**Comment by edeutsch** [comment]
yes, I see the same. This is usually either a copy paste error or the ARS instance that holds this UUID is having problems. The UUID appears the have the right number of digits, so probably the latter?

**Comment by MarkDWilliams** [comment]
It looks like there was an ARS outage at about midnight Eastern.  That is likely what caused the problems y'all had in retrieving results.  Apologies for the inconvenience; the server is back up now.

**Comment by andrewsu** [comment]
When running locally, this query returns 55 results.  But running it through the async endpoint via the ARS has surfaced a number of issues, which we are now working on:
* https://github.com/biothings/BioThings_Explorer_TRAPI/issues/313
* https://github.com/biothings/BioThings_Explorer_TRAPI/issues/349
* https://github.com/biothings/BioThings_Explorer_TRAPI/issues/348
